.github/workflows/ci.yml

```diff
@@ -103,34 +113,39 @@ jobs:
           CC: clang-10
           CXX: clang++-10
           RUN_ALL_TESTS: 1
+          RUN_MEMLEAK_TEST: 1
```

Comment:
what do we gain from running this test on every llvm version? This test seems to have a somewhat high overhead as we have to recompile bpftrace, think it be better to have the memleak test as a separate job/workflow.

Reply:
LLVM is known to contain memory leaks, so this way we know if they affect us on particular versions when we add a new feature. But I agree that the overhead is quite high which is probably more important than the benefit.